src/tribler/core/components/popularity/rendezvous/db/orm_bindings/certificate.py

```diff
@@ -0,0 +1,14 @@
+from pony.orm import Required, db_session
+
+
+def define_binding(db):
+    class RendezvousCertificate(db.Entity):
+        public_key = Required(bytes, index=True)
+        counter = Required(int)
+
+        @classmethod
+        @db_session
+        def get_count(cls, pk: bytes) -> int:
+            return RendezvousCertificate.get(public_key == pk).count()
```

Comment:
It looks like the method is currently not used. It can probably be deleted.

If you want to keep the method, it should be fixed, as the code is incorrect: `RendezvousCertificate.get(...)` call returns a single certificate object, and a single certificate object does not have the `count()` method.

The correct code (if it is necessary) should probably looks like:

```python
        def get_count(cls, pk: bytes) -> int:
            certificate = RendezvousCertificate.get(public_key=pk)
            return 0 if certificate is None else certificate.counter
```